### PR TITLE
style: add styles to homepage

### PR DESCRIPTION
**Pull request by connerdodge**
Addresses issue #21 

Adds homepage Title and Description styles

### CODE REVIEW

**Review by fridaland — CHANGES_REQUESTED** [review]
@connerdodge this is looking great! 
Perhaps it wasn't totally clear on the issue- but would you mind adjusting the container for the headline and description to match the mockup below? 
Screen Shot 2020-10-23 at 2 05 31 PM
The only thing that seems to be missing is floating the elements as pictured above.

**Comment by connerdodge** [comment]
@fridaland Let me know if the recent commit addresses the issue.

File: src/styles/index.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    font-weight: bold;
+    font-size: 48px;
+    color: #2C358F;
+    margin: 0 20px 0 60px;
+}
+
+.headline-description {
+    font-size: 20px;
+    color: #000000;
+    max-width: 450px;
```
Comment (fridaland) [inline src/styles/index.css]: It would be good to increase the width a bit on the description
```suggestion
    width: 800px;
```

**Review by fridaland — CHANGES_REQUESTED** [review]
Just one small comment on the description width, otherwise this looks great!

**Review by fridaland — APPROVED** [review]
Beautiful!

**Pull request merged**